cmd/snap/cmd_info.go

```diff
@@ -163,27 +163,68 @@ func coalesce(snaps ...*client.Snap) *client.Snap {
 	return nil
 }
 
-// formatDescr formats a given string (typically a snap description)
+func rTrimRightSpace(text []rune) []rune {
+	j := len(text)
+	for j > 0 && unicode.IsSpace(text[j-1]) {
+		j--
+	}
+	return text[:j]
+}
+
+func rLastIndexSpace(a []rune) int {
+	for i := len(a) - 1; i >= 0; i-- {
+		if unicode.IsSpace(a[i]) {
+			return i
+		}
+	}
+	return -1
+}
+
+// this is _wrong_ for much of unicode (because the width of a rune on the
```

Comment:
I saw a page-long function that was good for good chunk of Unicode. Perhaps we can come up with something that is better than nothing?